apps/snitch_core/lib/core/data/schema/card.ex

```diff
@@ -38,10 +35,10 @@ defmodule Snitch.Data.Schema.Card do
     |> validate_number(
       :month,
       less_than_or_equal_to: 12,
-      greater_than_or_equal_to: @current_month
+      greater_than_or_equal_to: DateTime.utc_now() |> Map.fetch!(:month)
```

Comment:
Though this is correct, a more readable and slightly efficient way if to get the current year and month together at the top,
```elixir
def changeset(card, params, :create) do
  %{year: current_year, month: current_month} = DateTime.utc_now()
  ...
end
```